client/components/bcaap-form-sections/application-dashboard.tsx

```diff
@@ -113,22 +136,31 @@ export const ApplicationDashboard: React.FC<any> = () => {
         <>
           <div className='w-full border py-4 px-8 mb-2'>
             Filter By:
-            <div className='grid grid-cols-3 gap-1'>
+            <div className='grid grid-cols-4 gap-1'>
               <input
                 type='text'
                 className='bg-white rounded border border-gray-300 text-gray-900 text-sm  focus:ring-blue-500 focus:border-blue-500 block w-full pr-10 p-2.5  dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500'
-                placeholder={'Facility Name'}
-                onChange={e => setState(p => ({ ...p, searchFacilityName: e.target.value }))}
-                value={searchFacilityName}
+                placeholder='Application Type'
+                onChange={e => setState(p => ({ ...p, searchApplicationType: e.target.value }))}
+                value={searchApplicationType}
               />
+
               <input
                 type='text'
                 className='bg-white rounded border border-gray-300 text-gray-900 text-sm  focus:ring-blue-500 focus:border-blue-500 block w-full pr-10 p-2.5  dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500'
-                placeholder={'Confirmation ID'}
+                placeholder='Confirmation ID'
                 onChange={e => setState(p => ({ ...p, searchConfirmationID: e.target.value }))}
                 value={searchConfirmationID}
               />
 
+              <input
+                type='text'
+                className='bg-white rounded border border-gray-300 text-gray-900 text-sm  focus:ring-blue-500 focus:border-blue-500 block w-full pr-10 p-2.5  dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500'
```

Comment:
maybe adding this class to a single const would clean it up a bit, seems to be the same for all 3 inputs